src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -109,11 +109,14 @@ void Background_Loaded(object sender, RoutedEventArgs e)
 
         void Wndhost_LayoutUpdated(object? sender, EventArgs e)
         {
-            var locationFromScreen = _bckgnd.PointToScreen(_zeroPoint);
-            var source = PresentationSource.FromVisual(_wndhost);
-            var targetPoints = source.CompositionTarget.TransformFromDevice.Transform(locationFromScreen);
-            Left = targetPoints.X;
-            Top = targetPoints.Y;
+            if (PresentationSource.FromVisual(_bckgnd) != null)
```

Comment:
To match the style just below, I'd do this.

```suggestion
            if (PresentationSource.FromVisual(_bckgnd) == null)
                return;
```